src/WebJobs.Extensions.OpenAI.CosmosDBSearch/CosmosDBSearchConfigOptions.cs

```diff
@@ -1,16 +1,67 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-namespace Microsoft.Azure.WebJobs.Extensions.OpenAI.CosmosDBSearch;
-
-/// <summary>
-/// Open AI Configuration Options used for reading host.json values.
-/// </summary>
-public class CosmosDBSearchConfigOptions
-{
-    // Number of dimensions for vector similarity. The maximum number of supported dimensions is 2000.
-    public int VectorSearchDimensions { get; set; } = 1536;
-
-    // This integer is the number of clusters that the inverted file (IVF) index uses to group the vector data
-    public int NumLists { get; set; } = 1;
-}
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenAI.CosmosDBSearch;
+
+/// <summary>
+/// Open AI Configuration Options used for reading host.json values.
+/// </summary>
+public class CosmosDBSearchConfigOptions
+{
+    // Database name for the vector store
+    public string DatabaseName { get; set; } = "openai-functions-database";
+
+    // Index name for your vector store collection
+    public string IndexName { get; set; } = "openai-functions-index";
+

```

Comment:
any change expected in connection string to supply any of these values apart from default connection string?

Reply:
For now only connection string should be good.